### DISCUSSION TITLE
Welcome to Relate Post Benchmark Discussions!

### DISCUSSION THREAD

**Discussion by zupat** [jinyus/related_post_gen#296] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Imperatorn** [comment]
I would just like to say I enjoy reading the solutions in various languages.

There's almost always a trade off between memory and speed, LOC and readability etc etc.
It's hard to make these kinds of things fair to everyone, because all conditions are not easy to even out.

But, it's still fun, and I would like to thank you for creating this.
